src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/request/AddOrUpdateQuestionWithResponses.kt

```diff
@@ -26,6 +26,13 @@ data class AddOrUpdateQuestionWithResponses(
   )
   @field:Size(min = 1)
   val question: String,
+  @Schema(
+    description = "The question text as seen by the user at the point of entry",
+    requiredMode = Schema.RequiredMode.NOT_REQUIRED,
```

Comment:
What does `requiredMode` do? If the field is mandatory should this be `NOT_REQUIRED`? 🤔